ci/azure-rustfmt.yml

```diff
@@ -0,0 +1,17 @@
+jobs:
+# Check formatting
+- job: ${{ parameters.name }}
+  displayName: Check rustfmt
+  pool:
+    vmImage: ubuntu-16.04
+  steps:
+    - template: azure-install-rust.yml
+      parameters:
+        # Pin the version of Rust in case rustfmt changes.
+        rust_version: 1.32.0
```

Comment:
I feel we should track stable, in the sense that most users will be on latest stable, so in theory the whole repo should stay somewhat up to date.